libraries/botbuilder-dialog-prompts/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@botbuildercommunity/dialog-prompts",
+  "version": "0.1.0",
+  "description": "A collection of dialog prompts for the Microsoft Bot Framework that implements Microsoft's text recognizer suite.",
+  "main": "./lib/index.js",
+  "typings": "./lib/index.d.ts",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "postinstall": "tsc -p .",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/BotBuilderCommunity/botbuilder-community-js.git"
+  },
+  "keywords": [
+    "chatbots",
+    "bot framework",
+    "bot builder",
+    "dialogs",
+    "prompts",
+    "recognizers"
+  ],
+  "contributors": [
+    "Bot Builder Community",
+    "Michael Szul <michael@szul.us>"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/BotBuilderCommunity/botbuilder-community-js/issues"
+  },
+  "homepage": "https://github.com/BotBuilderCommunity/botbuilder-community-js#readme",
+  "dependencies": {
+    "@microsoft/recognizers-text-number": "^1.1.3",
+    "@microsoft/recognizers-text-number-with-unit": "^1.1.3",
+    "@microsoft/recognizers-text-sequence": "^1.1.3",
+    "botbuilder": "^4.0.6",
+    "botbuilder-core": "^4.0.6",
+    "botbuilder-dialogs": "^4.0.6",
+    "mocha": "^5.2.0"
```

Comment:
I believe it is best practice to put `mocha` in `devDependencies`.

Reply:
Yeah, I need to cleanup the packages a bit. I tend to shove everything in dependencies, and I know that's not best practices. Once I get tests running, I'll likely move that out.